share/docker/Dockerfile

```diff
@@ -66,8 +66,8 @@ RUN apt-get -qq update && \
     maven \
     nodejs \
     perl \
-    php5.6 \
-    php5.6-gmp \
+    php7.1 \
```

Comment:
Maybe move the PHP version to an `ENV` to have it in one single place?

Reply:
Thanks for the review @Fokko! Just updated the PR.